File: 3014 - Minimum Number of Pushes to Type Word I/Method 1/solution.java
class Solution {
    public int minimumPushes(String word) {
        int n = word.length();
        int d = n / 8;
        int rem = n % 8;
        int sum = 0;
        for(int i = 1; i<=d; i++) {
            sum = sum + i * 8;
        }
        return sum + rem * (d+1);
    }
}

File: 3014 - Minimum Number of Pushes to Type Word I/Method 1/solution.js
/**
 * @param {string} word
 * @return {number}
 */
var minimumPushes = function(word) {
    let n = word.length;
    let d = Math.floor(n / 8);
    let rem = n % 8;
    let sum = 0;
    for(let i = 1; i<=d; i++) {
        sum = sum + i * 8;
    }
    return sum + rem * (d+1);
};